### ISSUE TITLE
is there something like isort for js?

### ISSUE THREAD

**Issue by surchs** [neurobagel/annotation_tool#354]
I notice we sometimes have diffs to re-sort object keys and such things. Is there maybe some tool we can configure to go through the whole codebase once to make the appropriate changes, and to then keep on top of this - e.g. as part of a pre-commit check?

**Comment by jarmoza** [comment]
I believe we can do this by adding the ["sort-keys" rule](https://eslint.org/docs/latest/rules/sort-keys) to the eslint rc file, and then by installing [this plugin](https://www.npmjs.com/package/eslint-plugin-sort-keys).

**Comment by surchs** [comment]
Cool! I'm in favour of adopting the eslint rule to keep things consistent. I'm less clear about the tool, it looks not very widely adopted / maintained. I could imagine that having the rule enabled and configured to show up in the IDE (and during pre-commit) might be sufficient here. wdyt?

**Comment by jarmoza** [comment]
'sort-keys' definitely works. I tried it out before I commented earlier. I can see it being annoying to the coder to do it manually. I'm alright with it though myself.

That said, I don't know if `eslint --fix` will alphabetize it. If so, then that's even better.

**Comment by surchs** [comment]
Yes there is something like `isort` in js, it's called https://www.npmjs.com/package/eslint-plugin-import . I'll make an issue to implement
